### PR TITLE
Add golangci-lint

### PR DESCRIPTION
**Pull request by gleich**
golangci-lint is a CLI tool that takes a ton of golang linters and runs them all at once on your project. It is very popular in the go community and enforces some best practices when working with go. I believe that using tools such as golangci-lint and following go's best practices is crucial to the long-term health of haas. Things are simply now, let's not screw ourselves later down the road.

In order to work with this, we need to conform to the errcheck linter. The errcheck linter makes sure all functions that return an `error` get checked. As seen in the first commit I made there are a number of places where this needed to be added. A lot of these places were in goroutines that I am not 100% sure the context for. For those cases, I simply log the error to the console instead of causing a fatal log.

Signed-off-by: Matthew Gleich <git@mattglei.ch>

### CODE REVIEW

**Review by veritem — APPROVED** [review]
LGTM!

File: pkg/api/apps/logs.go [hunk 1]
```diff
@@ -143,6 +147,10 @@ func handleGETLogs(c *gin.Context) {
 			break
 		}
 
-		ws.WriteJSON(logs)
+		err = ws.WriteJSON(logs)
+		if err != nil {
+			c.JSON(500, gin.H{"status": "error", "message": "Failed to convert logs to JSON"})
```
Comment (anirudhb) [inline pkg/api/apps/logs.go]: Nit: We can't really do this here since it's a WebSocket connection at this point

File: pkg/biller/biller.go [hunk 2]
```diff
@@ -109,6 +119,7 @@ type BillerEvent struct {
 }
 
 var billerOutputs map[uint]map[chan decimal.Decimal]struct{} = make(map[uint]map[chan decimal.Decimal]struct{})
+
```
Comment (anirudhb) [inline pkg/biller/biller.go]: Shouldn't be a new line here, maybe between the next two lines

**Review by anirudhb — CHANGES_REQUESTED** [review]
lgtm. Thanks for the catch, nits

**Review by anirudhb — APPROVED** [review]

**Pull request merged**